DataConnectors/Beyond Security beSECURE.json

```diff
@@ -0,0 +1,128 @@
+  "graphQueries": [
+    {
+      "metricName": "Total data received",
+      "legend": "Beyond Security beSECURE",
+      "baseQuery": "union beSECURE_Audit_CL, beSECURE_ScanEvent_CL, beSECURE_ScanResults_CL"
+    }
+  ],
+  "sampleQueries": [
+	{
+	"description" : "CVSS Score Aggregated",
+	"query": "beSECURE_ScanResults_CL  | summarize count() by cvss_score_s"
+	},
+	{
+	  "description" : "Risk Aggregated",
+	  "query": "beSECURE_ScanResults_CL  | summarize count() by risk_name_s"
+	},
+	{
+	  "description": "Host Risk Aggregated",
+	  "query": "beSECURE_ScanResults_CL  | summarize count() by destination_host_s, risk_name_s"
+	},
+	{
+	  "description" : "Top 10 most vulnerable Hosts",
+	  "query": "beSECURE_ScanResults_CL  | \n where (risk_value_s == 8 or risk_value_s == 4) | \n summarize count() by destination_host_s, risk_value_s | \n order by risk_value_s, count_ | \n limit 10"
+	},
+  {
+    "description" : "Failed login",
+    "query": "beSECURE_Audit_CL | \n where audit_event_s == 'Login failed' "
+  },
+    {
+      "description": "Completed Scans",
+      "query" : "beSECURE_ScanEvent_CL | \n where scan_event_s == 'Scan Complete' | \n summarize count()"
+    }
+    ,
+    {
+      "description": "Failed to Complete Scans",
+      "query" : "beSECURE_ScanEvent_CL | \n where scan_event_s == 'Scan Failed' | \n summarize count()"
+    }
+
+  ],
+  "dataTypes": [
+    {
+      "name": "beSECURE_ScanResults_CL",
+      "lastDataReceivedQuery": "beSECURE_ScanResults_CL \n | summarize Time = max(TimeGenerated)\n      | where isnotempty(Time)"
+    },
+	  {
+      "name": "beSECURE_ScanEvent_CL",
+      "lastDataReceivedQuery": "beSECURE_ScanEvent_CL \n | summarize Time = max(TimeGenerated)\n      | where isnotempty(Time)"
+    },
+	  {
+      "name": "beSECURE_Audit_CL",
+      "lastDataReceivedQuery": "beSECURE_Audit_CL \n | summarize Time = max(TimeGenerated)\n      | where isnotempty(Time)"
+    }
+
+  ],
+  "connectivityCriterias": [
+    {
+      "type": "IsConnectedQuery",
+      "value": [
+        "beSECURE_ScanResults_CL \n | summarize LastLogReceived = max(TimeGenerated) \n | project IsConnected = LastLogReceived > ago(30d)"
+      ]
+    }
+  ],
+  "availability": {
+    "status": 1
+  },
+  "permissions": {
+    "resourceProvider": [
+      {
+        "provider": "Microsoft.OperationalInsights/workspaces", 
+                       "permissionsDisplayText": "write permission is required.", 
+                       "providerDisplayName": "Workspace", 
+                       "scope": "Workspace", 
+                       "requiredPermissions": { 
+                       "write": true, 
+                       "delete": true 
+					}
+      }
+    ],
+    "customs": [
+    ]
+  },
+  "instructionSteps": [
+    {
+      "title": "",
+      "description": ">**NOTE**: This data connector depends on a parser based on a Microsoft Kusto Function to work as expected.",

```

Comment:
I don't think there is a parser dependency for this connector, right? If not, please delete this NOTE. Basically the instruction step for this instruction can go away which is title, description as well as instructions[].

If there's indeed some parser dependency please link that parser explicitly in this data connector so that that dependency is completely communicated.

Reply:
This dependency came from Symantec, which has similar requirement and listed this as a dependency - not sure why - I saw that Kusto is the "query" language used by Sentinel, anyway, removed

Reply:
Symantec is a syslog data connector and there's a specific parser built that needs to be included while ingesting data. This one doesn't seem to have a parser that's why the step is not needed, as causes confusion. Thanks, for removing.